README.md

```diff
@@ -0,0 +1,86 @@
+1. Under the "Upcoming events" section locate an event [probably the "Seeded Test Event"].
+1. Under that event's details click the "Organizer Console" button (or find a different event if that button is not present).
+1. Scroll to the bottom of the page.
+1. Observe that within the "Operating System Breakdown" table each row has a count and a percentage of total student attendees with that operating system.
+1. Observe that for each row where the percentage is not exact it is instead listed with a very large number of digits, e.g. "10.204081632653061%". [Note: You may not see this because the seeded event is random.]
+
+#### 3. Bug Finding
+Now that we have a set of repro steps that we can follow to trigger the bug, we can start using those steps to pin-point what code is involved.
+
+Because this bug involves the UI primarily (something is being displayed to the user in an undesirable way), we should start with our view code. It's possible that the underlying issue is actually elsewhere in the code base, but the view code is a good place to start.
+
+We need to identify which view template is responsible for the Operating System Breakdown table. One way of figuring this out would be to look at the URL (e.g. "http://localhost:3000/events/1/organizer_tools") and match that up with a particular route, then use that to find the associated controller action and eventually view template.
+
+However, a simpler option would be to search through the whole project for the text "Operating System Breakdown", because that's the literal text for the header on this table. You can search through your whole project in Atom by using Cmd + Shift + F.
+
+Either option should eventually have you end up at the file `app/views/events/organizer_tools/_operating_system_breakdown.erb`. This partial view template is responsible for just the OS table section of the organizer tools page.
+
+From here we can find that line 20 is the part where we output the contents of the "Count" column in the table, including the percentage.
+
+#### 4. Bug Fixing
+We've located the code that is most likely involved with our bug:
+```erb
+<%= count %> (<%= ((count.to_f / total) * 100) %>%)
+```
+
+Now we need to figure out how to modify it to fix the issue. There are several ways of going about this, depending on exactly what results we want. In this example I'm going to specify that we want to only output 2 decimal points for the percentage. The simplest way to solve that is by using [Ruby's format string functionality](https://ruby-doc.org/core-2.2.0/String.html#25-method):
+```erb
+<%= count %> (<%= '%.2f' % ((count.to_f / total) * 100) %>%)
+```
+
+With that change in place, we can reload the page and check if our fix worked. Since this is a UI issue we don't have any automated tests that we can run to verify the fix, but with all of the other bugs in the activity you should see that at least one of the failing tests is working again if the bug has been fixed properly.
+
+## Bugs
```

Comment:
It seems like there is a big leap between the end of this explanation and then the beginning of the bugs list. Maybe just a bit more verbiage towards the bottom here prior to introducing the bugs?

Reply:
:+1: